### DISCUSSION TITLE
Is it possible to switch the built-in open/save file dialog to use Windows' native file picker?

### DISCUSSION THREAD

**Discussion by winstxnhdw** [freeplane/freeplane#1591] [Help]
I am interested to develop an extension to Freeplane to do this but I just want to make sure if it's even possible or this is a feature that does not already exists.

**Comment by hug956** [comment]
in my opinion, offering the native Windows file selector requires also offering the equivalent on all platforms (Linux, IOS, BSD, etc.).
perhaps adding only the address bar in this selector, compatible with all platforms, will be sufficient for most uses.

**Reply by winstxnhdw** [reply]
I am not trying to build for cross-platform support. I want to build only for Windows with the Windows file selector. I am also not looking at the developers to do it for me but give me a direction I can look at to implement this myself.
